### ISSUE TITLE
Generate ApplicationLayout spec

### ISSUE THREAD

**Issue by cllns** [hanami/hanami#735]
We generate `ApplicationLayout` for new apps, but not an associated spec.

Here's what it could look like, but probably without the  `it` block (shown for example), located at: `spec/web/views/application_layout_spec.rb`
```
require "spec_helper"
require_relative "../../../apps/web/views/application_layout"

describe Web::Views::ApplicationLayout do
  let(:layout)   { Web::Views::ApplicationLayout.new(template, {}) }
  let(:rendered) { layout.render }
  let(:template) { Hanami::View::Template.new("apps/web/templates/application.html.slim") }

  it "has #site_name" do
    layout.site_name.must_equal "My Hanami Web App"
    rendered.must_include "My Hanami Web App"
  end
end
```

Note: this is a suggestion for `v1.1`, **not** `v1.0` :)

**Comment by jodosha** [comment]
Marked as enhancement given the general consensus.

**Comment by akhramov** [comment]
@jodosha, may I take this one?

Thanks!

**Comment by cllns** [comment]
@akhramov go for it!

**Comment by cllns** [comment]
Closed by #774 🎉

**Comment by thiagopecanha** [comment]
Maybe the it block could it be something like
'''
describe "an example" do
  it "is implemented but waiting" do
    pending("something else getting finished") do
      raise "this is the failure"
    end
  end
end
'''
